This patch prevents a warning message from being generated when the input file
 is not an xccdf results file.

--- openscap-0.9.7/xsl/security-guide.xsl.orig    Tue Oct 29 11:52:16 2013
+++ openscap-0.9.7/xsl/security-guide.xsl    Tue Nov 12 15:37:21 2013
@@ -256,5 +256,7 @@
   </row>
 </xsl:template>

+<!-- override warn-unresolved for guide generation -->
+<xsl:template name='warn-unresolved'/>

 </xsl:stylesheet> 
